projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -310,6 +315,22 @@ private function load_value_from_callback( $item, $callback, $default = '' ) {
 		return $default;
 	}
 
+	/**
+	 * Loads any extra data for a task, calling the 'extra_data_callback' callback if it exists,
+	 * or an empty array if there is no callback.
```

Comment:
With the most recent update, we return `null` instead of an empty array.
```suggestion
	 * or null if there is no callback or the callback returns an empty array.
```